grpc/Cargo.toml

```diff
@@ -28,4 +28,5 @@ dns = ["dep:hickory-resolver"]
 allowed_external_types = [
     "tonic::*",
     "futures_core::stream::Stream",
-]
\ No newline at end of file
+    "protobuf::codegen_traits::Message",
```

Comment:
We need to add a todo here that we want to remove this. This should all be abstracted by the new protobuf crate. (aka its exposed as a Codec rather than needing the grpc crate to know about a message trait).